### PR TITLE
Fix cardlinks

### PR DESCRIPTION
**Pull request by Caruki**

### CODE REVIEW

**Comment by JuleFloorson** [comment]
looks nice :)

File: src/components/CardExample.js [hunk 1]
```diff
@@ -1,24 +1,44 @@
 import React from 'react';
 import styled from '@emotion/styled';
+// import isPropValid from '@emotion/is-prop-valid';
```
Comment (marwinburesch) [inline src/components/CardExample.js]: 🧐 ooh a comment

File: src/components/CardExample.js [hunk 2]
```diff
@@ -1,24 +1,44 @@
...
+    width: 90%;
+  }
+`;
+
+//   {
+//   shouldForwardProp: prop => isPropValid(prop) && prop === 'as'
+// })(
+//   props => ({
+//   as: 'a'
+// }))`
```
Comment (marwinburesch) [inline src/components/CardExample.js]: 😑

Comment (Caruki) [reply src/components/CardExample.js]: I realized *after* merging 😞

File: src/components/CardExample.js (hunk 1 above)
Comment (Caruki) [reply src/components/CardExample.js]: I know 😞

**Pull request merged**